### PR TITLE
Add builds without installing the optional dependencies

### PR DESCRIPTION
**Pull request by bsipocz**
While this is somewhat tested via appveyor, I think it's worth having at least one of these builds for travis, too.

### CODE REVIEW

**Comment by astrofrog** [comment]
The ``fits_compare`` decorator needs to be changed to ``array_compare``. We'll also need to explicitly specify the file format in each decorator, but I think I want to add a flag to make that a global option, so maybe I can fix that first.

**Comment by bsipocz** [comment]
Do you want me to do the decorator name change in this PR?

**Comment by astrofrog** [comment]
@bsipocz - yes please!

**Comment by astrofrog** [comment]
I'm working on the default file format option

**Comment by astrofrog** [comment]
@bsipocz - once https://github.com/astrofrog/pytest-arraydiff/pull/2 is merged, you can use that flag here to set the default file format to FITS

**Comment by bsipocz** [comment]
OK. I have 9 failures locally, and I stopped appveyor, but couldn't do it for travis.

**Comment by astrofrog** [comment]
@bsipocz https://github.com/astrofrog/pytest-arraydiff/pull/2 is merged so you should now be able to add ``--arraydiff-default-format=fits`` in ``setup.cfg``

**Comment by bsipocz** [comment]
It passes now locally, so should be fine here, too.

**Comment by astrofrog** [comment]
@bsipocz - thanks! :)

File: setup.cfg [hunk 1]
```diff
@@ -11,7 +11,7 @@ show-response = 1
 minversion = 2.2
 norecursedirs = build docs/_build
 doctest_plus = enabled
-addopts = --fits
+addopts = --arraydiff-default-format=fits
```
Comment (astrofrog) [inline setup.cfg]: @bsipocz - ah, you need to also make sure ``--arraydiff`` is included here on its own before the ``--arraydiff-default-format`` - sorry for not spotting that sooner

Comment (bsipocz) [inline setup.cfg]: oops, sorry.

**Comment by astrofrog** [comment]
@bsipocz - ok, so now the failures are because arraydiff only works with arrays and we should return arrays instead of HDUs. However, I think it'd be nice to continue to support HDUs, so if the format is FITS we should allow HDUs to be returned (which requires a change in arraydiff)

**Comment by astrofrog** [comment]
@bsipocz - can you rebase this to get it to rebuild now with the new pytest-arraydiff changes? You'll need to keep installing the dev version for now to get the latest HDU-related changes

**Comment by bsipocz** [comment]
Rebased (though it was already on top of master, restarting the builds should have worked I guess).

**Comment by bsipocz** [comment]
something is now very badly broken, can look at it after dinner...

**Comment by astrofrog** [comment]
It might be due to a change from ``baseline`` to ``reference`` for the default reference directory?

**Comment by astrofrog** [comment]
Thanks @bsipocz!

**Pull request merged**